CFC_WebApp/clients/socialmedia/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "socialmedia",
+  "start_date": "2014/10/13",
+  "end_date": "2015/06/01",
+  "key": "dc23ad22-bc78-4fd6-9adf-ca2b04505916",
+  "client_settings": {
+    "result_url": "https://e-mission.eecs.berkeley.edu/socialmedia",
```

Comment:
You do not want this. Setting this is intended to display https://e-mission.eecs.berkeley.edu/socialmedia in the result screen instead of 
https://e-mission.eecs.berkeley.edu/compare.

And you haven't even defined the https://e-mission.eecs.berkeley.edu/socialmedia endpoint.

I might have disabled the client settings code in the apps - that is pretty much dead code at this point. But at any rate, just leave this as "compare".